doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -352,6 +354,25 @@ B<label>. The return value is the label length. The padding mode
 must have been set to B<RSA_PKCS1_OAEP_PADDING>. The resulting pointer is owned
 by the library and should not be freed by the caller.
 
+B<RSA_PKCS1_WITH_TLS_PADDING> is used when decrypting an RSA encrypted TLS
+pre-master secret in a TLS ClientKeyExchange message. It is the same as
+RSA_PKCS1_PADDING except that it additionally verifies that the result is the
+correct length and the first two bytes are the protocol version initially
+requested by the client. If the padding checks fail then decryption will still
+appear to succeed but a random TLS premaster secret will be returned instead
+unless the encrypted content is publicly invalid (in which case the decryption
+will fail as normal). This padding mode accepts two parameters which can be set
```

Comment:
This is hard to parse

Reply:
I reworded it a little and tried to break it up into two sentences. Hopefully its better.